etc/helm/pachyderm/templates/pgbouncer/deployment.yaml

```diff
@@ -79,6 +79,23 @@ spec:
         image: bitnami/pgbouncer:1.16.1-debian-10-r82
         imagePullPolicy: IfNotPresent
         name: pg-bouncer
+        livenessProbe:
+          failureThreshold: 3
+          exec:
+            command:
+              - "/bin/sh"
+              - "-c"
+              - psql postgresql://$POSTGRESQL_USERNAME:$POSTGRESQL_PASSWORD@127.0.0.1:$PGBOUNCER_PORT/ --tuples-only --command="SELECT 1;" | grep -q "1"
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        readinessProbe:
```

Comment:
Not sure how applicable to pgbouncer this is, but postgres ships with `pg_isready` to see if postgres is ready: https://www.postgresql.org/docs/current/app-pg-isready.html

Reply:
yeah it isn't the the pg_bouncer image

Reply:
fair enough!